articles/libraries/chemistry/concepts/algorithms.md

```diff
@@ -29,19 +29,19 @@ Note that if $e^{-i H t}$ were an ordinary exponential then the error in this ap
 This error occurs because $e^{-iHt}$ is an operator exponential and as a result there is an error incurred when using this formula due to the fact that the $H_j$ terms do not commute (*i.e.*, $H_j H_k \ne H_k H_j$ in general).
 
 If $t$ is large, Trotter–Suzuki formulas can still be used to simulate the dynamics accurately by breaking it up into a sequence of short time-steps.
-Let $r$ be the number of steps taken in the time evolution.
+Let $r$ be the number of steps taken in the time evolution, so each time step runs for time $t/r$. 
 Then, we have that
 $$
-    e^{-i\sum_{j=1}^m H_j t} =\left(\prod_{j=1}^m e^{-iH_j t/r}\right)^r + O(m^2 t^2/r),
+    e^{-i\sum_{j=1}^m H_j t} =\left(\prod_{j=1}^m e^{-iH_j t/r}\right)^r + O(m^3 t^2/r),
```

Comment:
This looks like the first-order Trotter–Suzuki formula, which definitely has quadratic error scaling in 𝑡, but I don't have the scaling with 𝑚 memorized, sorry. Did this also come from https://iopscience.iop.org/article/10.1088/1367-2630/aab1ef/pdf? https://arxiv.org/abs/1211.4945 may be another helpful reference as well.

Reply:
Whoops, yeah I took that change out. If I recall correctly, the m^3 falls out from a generous upper bounding after repeatedly applying `e^{-iA} e^{-iB} = e^{-i(A+B)} + O(d^2)` all the way through `H_m` (d<1 the max spectral norm). But that's definitely far from optimal, so the m^2 dependence is probably the case.

Anyway, I added some clarification on the Suzuki recursion in the newest commit, along with the order subscripts for readability (seemed potentially confusing to use `U_2` when referring to the fourth-order). Basically just followed section C.1 on p.12 here https://arxiv.org/pdf/1711.10980.pdf. Let me know what you think.

Apologies for drawing on your time for this---the issue raised was just a simple missing factor, but when I looked at it a few more things seemed funky.